src/layout/landingPage/Body.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import { Flex, FlexItem, Title } from '@patternfly/react-core';
+import BodyTile from './BodyTile';
+import './styles/Body.scss';
+
+const Body = () => {
+  return (
+    <Flex className="ins-c-landing-body">
```

Comment:
I don't think you need to wrap this whole thing in a Flex container. Things should naturally place on their own and then if needed, we can use a gallery or something inside